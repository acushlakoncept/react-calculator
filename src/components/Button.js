/* eslint-disable react/prop-types */
import React from 'react';

export default function Button({ name }) {
  return <button type="button">{ name }</button>;
}
