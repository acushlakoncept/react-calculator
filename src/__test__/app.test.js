import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../components/Button';

it('renders correctly', () => {
  const tree = renderer.create(<Button name="AC" clickHandler={jest.fn()} />);
  expect(tree).toMatchSnapshot();
});
