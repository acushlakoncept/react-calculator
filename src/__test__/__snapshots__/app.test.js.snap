// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  onClick={[MockFunction]}
  type="button"
>
  AC
</button>
`;
